lib/validators/custom/confirmed.go

```diff
@@ -0,0 +1,14 @@
+package custom
+
+import (
+	"fmt"
+	"github.com/go-playground/validator/v10"
+)
+
+func ConfirmedValidator(fieldLevel validator.FieldLevel) bool {
+	field := fieldLevel.Field().String()
```

Comment:
```suggestion
	value := fieldLevel.Field().String()
```

Frome my understanding, this is not the field, but its value, no?